test/test_nested_protocols.py

```diff
@@ -1,8 +1,5 @@
 """Test that various test protocols are executed correctly, using a TestOdeModel."""
```

Comment:
I think most of the `TestOdeModel` uses have now left this file!

Reply:
Updated it, thanks!